README.md

```diff
@@ -4,13 +4,21 @@ CFPB's internal chat bot built on [Hubot](http://hubot.github.com).
 
 ## Installation
 
+### Install Node.js
+
+1. `curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.25.2/install.sh | bash`
+1. `nvm install 0.10`
+1. `nvm use 0.10`
+
```

Comment:
No but in the past I've run into deployment issues using anything other than 0.10.x. The cause is likely a third-party script but I don't have the time nor desire to investigate.